src/cpu/flexcpu/simple_dataflow_thread.cc

```diff
@@ -122,6 +122,35 @@ SDCPUThread::activate()
 
 void
 SDCPUThread::advanceInst(TheISA::PCState next_pc)
+{
+    DPRINTF(SDCPUThreadEvent, "Request issue for %s. outstanding %p\n",
+            next_pc, squashIssueFlag);
+
+    squashIssueFlag = new bool(false);
```

Comment:
Could just make this a `shared_ptr<bool>`, since it's basically a pointer that's shared between SDCPUThread and the two lambdas using it here. When each reference dies it will clean up without having to manually do deletes here.

Reply:
Sure. That makes sense.